Cargo.toml

```diff
@@ -17,8 +17,7 @@ log = "^0.4"
 futures = "0.3"
 dirs = "3.0.1"
 tokio-socks = "0.5"
-reqwest = { version = "0.11", features = ["socks"] }
-clap = { version = "4.3.19", features = ["derive"] }
+clap = { version = "3.2.22", features = ["derive"] }
```

Comment:
Any reason to downgrade to v3.*.* from v4.*.*

Reply:
V4 Doesn't have the custom parse function capability. I don't know why they removed it, and couldn't identify an alternative.